x/tally/keeper/integration_helpers_test.go

```diff
@@ -0,0 +1,287 @@
+package keeper_test
+
+import (
+	"encoding/base64"
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/cometbft/cometbft/crypto/secp256k1"
+	vrf "github.com/sedaprotocol/vrf-go"
```

Comment:
This is old, right?
I don't think we should be adding any VRF library (yet).

Reply:
I just wanted to replicate the overlay code and saw that overlay uses vrf library to generate the stake, commit, and reveal proofs.